### PR TITLE
Grammar fixes on output circuits.

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
spellcheck: fixed grammar on output circuits
/:cl:

### CODE REVIEW

**Comment by DaxDupont** [comment]
I wouldn't capitalize Examining personally.

File: code/modules/integrated_electronics/subtypes/output.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 /obj/item/integrated_circuit/output/screen
 	name = "small screen"
-	desc = "This small screen can display a single piece of data, when the machine is examined closely."
+	desc = "Takes any data type as an input, and displays it upon Examining."
```
Comment (ShizCalev) [inline code/modules/integrated_electronics/subtypes/output.dm]: Lowercase.

File: code/modules/integrated_electronics/subtypes/output.dm [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 /obj/item/integrated_circuit/output/screen/medium
 	name = "screen"
-	desc = "This screen allows for people holding the device to see a piece of data."
+	desc = "Takes any data type as an input, and displays it upon Examining, and to adjacent beings when pulsed."
```
Comment (ShizCalev) [inline code/modules/integrated_electronics/subtypes/output.dm]: Lowercase. Also remove the first comma.

File: code/modules/integrated_electronics/subtypes/output.dm [hunk 3]
```diff
@@ -43,7 +43,7 @@
 
 /obj/item/integrated_circuit/output/screen/large
 	name = "large screen"
-	desc = "This screen allows for people able to see the device to see a piece of data."
+	desc = "Takes any data type as an input, and displays it upon examining, and to all nearby beings when pulsed."
```
Comment (ShizCalev) [inline code/modules/integrated_electronics/subtypes/output.dm]: ditto

File: code/modules/integrated_electronics/subtypes/output.dm [hunk 4]
```diff
@@ -54,7 +54,7 @@
 
 /obj/item/integrated_circuit/output/light
 	name = "light"
-	desc = "This light can turn on and off on command."
+	desc = "A basic light, can be toggled on/off when pulsed."
```
Comment (ShizCalev) [inline code/modules/integrated_electronics/subtypes/output.dm]: A basic light which can

File: code/modules/integrated_electronics/subtypes/output.dm [hunk 5]
```diff
@@ -96,7 +96,7 @@
 
 /obj/item/integrated_circuit/output/light/advanced
 	name = "advanced light"
-	desc = "This light can turn on and off on command, in any color, and in various brightness levels."
+	desc = "A light that takes a hexadecimal color value and a brightness value, and can be toggled on and off with a pulse."
```
Comment (ShizCalev) [inline code/modules/integrated_electronics/subtypes/output.dm]: `on/off` or `on and off`? consistency.

**Comment by Bawhoppen** [comment]
Yea, there's no reason it should be capitalized. At best you should bold it.

File: code/modules/integrated_electronics/subtypes/output.dm [hunk 6]
```diff
@@ -128,7 +128,7 @@
 
 /obj/item/integrated_circuit/output/text_to_speech
 	name = "text-to-speech circuit"
-	desc = "A miniature speaker is attached to this component."
+	desc = "Takes any string as an input, and will make the device say the string when pulsed."
```
Comment (ShizCalev) [inline code/modules/integrated_electronics/subtypes/output.dm]: Do these really need a comma?

**Comment by Iamgoofball** [comment]
Fair point

File: code/modules/integrated_electronics/subtypes/output.dm [hunk 7]
```diff
@@ -225,7 +223,7 @@
 
 /obj/item/integrated_circuit/output/video_camera
 	name = "video camera circuit"
-	desc = "This small camera allows a remote viewer to see what it sees."
+	desc = "Takes a string as a name and a boolean to determine whether it is on, and uses this to be a Camera, linked to the Research network."
```
Comment (ShizCalev) [inline code/modules/integrated_electronics/subtypes/output.dm]: Lowercase camera. Also this seems awkwardly worded.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/integrated_electronics/subtypes/output.dm (hunk 2 above)
Comment (Iamgoofball) [reply code/modules/integrated_electronics/subtypes/output.dm]: radical

File: code/modules/integrated_electronics/subtypes/output.dm (hunk 6 above)
Comment (Iamgoofball) [reply code/modules/integrated_electronics/subtypes/output.dm]: probably dont need the oxford comma here, but yeah

File: code/modules/integrated_electronics/subtypes/output.dm (hunk 7 above)
Comment (Iamgoofball) [reply code/modules/integrated_electronics/subtypes/output.dm]: it's a weird one to explain, yeah, but it's better than the previous one

**Comment by ShizCalev** [comment]
Please merge these all into one PR. Preferably #32789

File: code/modules/integrated_electronics/subtypes/output.dm (hunk 6 above)
Comment (Cyberboss) [reply code/modules/integrated_electronics/subtypes/output.dm]: yeah remove it please

File: code/modules/integrated_electronics/subtypes/output.dm (hunk 7 above)
Comment (Cyberboss) [reply code/modules/integrated_electronics/subtypes/output.dm]: lowercase camera and research

**Pull request merged**